demo/assets/js/carousel.js

```diff
@@ -0,0 +1,141 @@
+(function() {
+
+    const CONTENT_VIEW_PORT = 280;
+
+	const PANEL_WIDTH = 120;
+	const FIRST_PANEL_IDX = 1;
+	const LAST_PANEL_IDX = 8;
+    const BASE = document.querySelector("#carousel-container").getBoundingClientRect().width / 2 - (PANEL_WIDTH / 2);
+
+	const carouselArea = document.querySelector("#carousel-area");
+	const detailViewImg = document.querySelector("#detail-view-content > img");
+    const btnMoreInfo = document.querySelector("#btn-more-info");
+    const detailViewArea = document.querySelector("#detail-view-carousel");
+    
+    let scaleRatio = 1;
+
+	const imgSrcCache = [];
+	const boxElmCache = [];
+	Array.from(document.querySelectorAll("[data-boxId]")).forEach((elm) => {
+        const boxId = elm.dataset.boxid;
+        imgSrcCache[boxId] = elm.children[0].src;
+        boxElmCache[boxId] = elm;
+    });
+
+    const detailViewAxes = new eg.Axes({
+        axis: {
+            X: {
+                range: [-50, 0]
+            },
+            Y: {
+                range: [-100, 0]
+            }
+        },
+        bounce: [0, 0],
```

Comment:
The bounce option must be applied to the axis like this
```js
X: {
   range: [ -50, 0],
   bounce: [ 0, 0]
}
```
the default value of bounce is `[0,0]`. so you can ignore it.